chef/main-cookbooks/teracy/Berksfile

```diff
@@ -2,3 +2,5 @@
 source 'https://supermarket.chef.io'
 
 metadata
+
+cookbook "kubernetes-stack", git: "https://github.com/teracyhq-incubator/kubernetes-stack-cookbook.git", branch: "develop"
```

Comment:
use the latest release from Chef Supermarket instead

Reply:
@hoatle cookbook 'kubernetes-stack', '~> 0.1.1' !

Reply:
fixed!